src/services/gas-price/providers/blocknative.provider.ts

```diff
@@ -64,7 +67,13 @@ export default class BlocknativeProvider {
       }
 
       // gas price is in gwei
-      return gasPrice != null ? gasPrice.price * GWEI_UNIT : null;
+      return gasPrice != null
```

Comment:
nit: i'd prefer a traditional if statement here compared to a ternary for the purpose of readability

Reply:
yeah I see what you mean, its a bit hard to read in this instance - I've pushed a fix for it